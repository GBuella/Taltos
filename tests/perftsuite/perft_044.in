setboard K7/8/2n5/1n6/8/8/8/k6N b - - 0 1
echo K7/8/2n5/1n6/8/8/8/k6N b - - 0 1
perft 1
perft 2
perft 3
perft 4
perft 5
perft 6
