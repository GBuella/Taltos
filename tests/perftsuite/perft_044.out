K7/8/2n5/1n6/8/8/8/k6N b - - 0 1
17
54
835
5910
92250
688780
